source/Ocl/Converters/DefaultAttributeOclConverter.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Collections.Generic;
+
+namespace Octopus.Ocl.Converters
+{
+    public class DefaultAttributeOclConverter : OclConverter
+    {
+        public override bool CanConvert(Type type)
+            => OclAttribute.IsSupportedValueType(type);
```

Comment:
At the moment this doesn't handle enums - they get treated as blocks.

Reply:
👍 Will put up another PR for that